### ISSUE TITLE
Write unit tests for the rest of the LLM classes

### ISSUE THREAD

**Issue by sestinj** [continuedev/contribution-ideas#15]

**Comment by khareyash05** [comment]
Hey @sestinj I would love to take this one up. If this is open. Thanks

**Comment by sestinj** [comment]
@khareyash05 Sure that would be great! Is there anything you'd want me to clarify to help you get started?

**Comment by khareyash05** [comment]
> @khareyash05 Sure that would be great! Is there anything you'd want me to clarify to help you get started?

Yeah. Just one question. I see the lines being commented for the tests. Is their a particular reason they are commented or are we waiting for all llm classes tests to be written?

**Comment by khareyash05** [comment]
> @khareyash05 Sure that would be great! Is there anything you'd want me to clarify to help you get started?

I will let you know if i need something. Thanks
